16-gen-2012 20.18.16 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.refreshBuddies()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.refreshBuddies(SkypeBuddyRoster.java:67)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.reload(SkypeBuddyRoster.java:218)
	at it.uniba.di.cdg.skype.SkypeBackend.OnPropertyChange(SkypeBackend.java:706)
	at it.uniba.di.cdg.skype.SkypeListeners.OnPropertyChange(SkypeListeners.java:215)
	at com.skype.api.Skype.HandlePropertyChange(Skype.java:176)
	at com.skype.ipc.EventDispatcher.run(EventDispatcher.java:92)
16-gen-2012 20.18.16 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBuddyRoster.refreshBuddies()]
java.lang.NullPointerException
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.refreshBuddies(SkypeBuddyRoster.java:67)
	at it.uniba.di.cdg.skype.SkypeBuddyRoster.reload(SkypeBuddyRoster.java:218)
	at it.uniba.di.cdg.skype.SkypeBackend.OnPropertyChange(SkypeBackend.java:706)
	at it.uniba.di.cdg.skype.SkypeListeners.OnPropertyChange(SkypeListeners.java:215)
	at com.skype.api.Skype.HandlePropertyChange(Skype.java:176)
	at com.skype.ipc.EventDispatcher.run(EventDispatcher.java:92)
9-feb-2012 16.28.06 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.28.06 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.28.07 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.25 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.54 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.54 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
9-feb-2012 16.30.54 it.uniba.di.cdg.aspects.ExceptionLoggingAspect ajc$afterThrowing$it_uniba_di_cdg_aspects_ExceptionLoggingAspect$1$442fbe6f
INFO: Exception trace aspect [SkypeBackend.skypeCleanup()]
java.lang.NullPointerException
	at com.skype.ipc.RootObject.Close(RootObject.java:147)
	at com.skype.api.Skype.Close(Skype.java:226)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeCleanup(SkypeBackend.java:661)
	at it.uniba.di.cdg.skype.SkypeBackend.skypeConnect(SkypeBackend.java:377)
	at it.uniba.di.cdg.skype.SkypeBackend.connect(SkypeBackend.java:443)
	at it.uniba.di.cdg.skype.test.SkypeBuddyRosterTest.setUp(SkypeBuddyRosterTest.java:27)
	at junit.framework.TestCase.runBare(TestCase.java:132)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
